data/transactions/logic/opcodes.go

```diff
@@ -184,6 +189,36 @@ func opBranch() OpDetails {
 	return d
 }
 
+func multiOp(dispatch byte, name string, rootVersion uint64, children ...OpSpec) OpSpec {
+	ret := OpSpec{Opcode: dispatch, Name: name, OpDetails: opDefault()}
+	ret.childOps = children
+	ret.Version = rootVersion
+	return ret
+}
+
+func isMultiOp(spec *OpSpec) bool {
+	return spec.childOps != nil
+}
```

Comment:
I'm glad it's this simple. By making this check _before_ resorting to `getLeaf`, I'm sure it will be inlined and we won't have a slowdown. (Not sure about the Deprecated checking yet though.)